assets/src/scripts/components/graph/view/lithology.js

```diff
@@ -1,3 +1,5 @@
+import {setSelectedConstructionId, setSelectedLithologyId} from "../../well-log/state";
```

Comment:
I know this is an annoying nit, but for consistency could you put spaces around the imports?

```suggestion
 import { setSelectedConstructionId, setSelectedLithologyId } from '../../well-log/state';
```

Reply:
No Problem. Fixed.